Sources/Sentry/SentryEnvelope.m

```diff
@@ -95,6 +95,8 @@ - (instancetype)initWithEvent:(SentryEvent *)event
     // default. In any case in the envelope type it should be event. Except for transactions
     NSString *envelopeType = [event.type isEqualToString:SentryEnvelopeItemTypeTransaction]
         ? SentryEnvelopeItemTypeTransaction
+        : [event.type isEqualToString:SentryEnvelopeItemTypeFeedback]
```

Comment:
`m`: Again a test would be nice.

Reply:
yeah this is part of why i want to validate the envelope contents